v22.1/ui-hot-ranges-page.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Hot Ranges Page
+summary: The Hot Ranges page provides details about ranges receiving a high number of reads or writes.
+toc: true
+docs_area: reference.db_console
+---
+
+The **Hot Ranges** page of the DB Console provides details about ranges receiving a high number of reads or writes. These are known as "hot ranges".
```

Comment:
Could you treat hot ranges as a defined term and set it in italics instead of inside quotes?